src/shipit_signoff/shipit_signoff/policies.py

```diff
@@ -29,10 +40,24 @@ def check_whether_policy_can_be_signed(email, group_name, policy, existing_signa
     ))
 
     if not can_user_still_sign:
-        missing_signoffs = _calculate_missing_signoffs(policy, existing_signatures)
+        missing_signoffs = _calculate_missing_signoffs(
+            policy, existing_signatures)
         raise NoSignoffLeftError(email, group_name, policy, missing_signoffs)
 
 
+def check_whether_policy_can_be_unsigned(email, group_name, policy, existing_signatures):
+    is_user_allowed_to_sign = any((
```

Comment:
Possibly, but it's the same logic. The question is  "Are they mentioned in the policy?" If so, they can add/remove their own signature. Perhaps we should give it a more neutral name